test/functional/ProgramCallFunctional.js

```diff
@@ -96,31 +110,44 @@ describe('ProgramCall Functional Tests', () => {
 
         const program = new ProgramCall('QWCRSVAL', { lib: 'QSYS' });
 
-        const outBuf = [
-          [0, '10i0'],
-          [0, '10i0'],
-          ['', '36h'],
-          ['', '10A'],
-          ['', '1A'],
-          ['', '1A'],
-          [0, '10i0'],
-          [0, '10i0'],
-        ];
-
-        const errno = [
-          [0, '10i0'],
-          [0, '10i0', { setlen: 'rec2' }],
-          ['', '7A'],
-          ['', '1A'],
-        ];
-
-        program.addParam(outBuf, { io: 'out' });
-        program.addParam(66, '10i0');
-        program.addParam(1, '10i0');
-        program.addParam('QCCSID', '10A');
-        const paramValue = 'errno';
-
-        program.addParam(errno, { io: 'both', len: 'rec2', name: paramValue });
+        const outBuf = {
```

Comment:
The second test in [ProgramCallFunctional.js](https://github.com/IBM/nodejs-itoolkit/blob/master/test/functional/ProgramCallFunctional.js#L94) Does not seem useful. 
Its the exact same test as the first the only difference it adds name field for errno DS.

We should remove it and add other useful ProgramCall functional tests.

Reply:
yeah, I think the reason the name was originally in a variable was to test that it comes back, but it never did. Probably makes sense to just remove.